.gitignore

```diff
@@ -263,3 +263,4 @@ paket-files/
 
 # Visual Studio Code
 .vscode
+/samples/workload-install.ps1
```

Comment:
Is this specific to your local build machine, Vlad? I haven't seen a `workload-install` PowerShell script in our repo before.

Reply:
it is a Tizen script. each time I open the solution, this file is created. as this is a gitignore file, I don't see any problem adding this file.